docs/index.rst

```diff
@@ -41,4 +41,4 @@ See the `GitHub contributor page <https://github.com/pymc-devs/pymc-experimental
 .. toctree::
    :hidden:
 
-   api_reference
+   api_reference <https://www.pymc.io/projects/experimental/en/latest/api_reference.html>
```

Comment:
```suggestion
   api_reference
```

This should not be changed. We want this to generate a link to the api reference of the version for which the docs are being built. Changing that we'd be hardcoding the link to always point to latest while at the same time excluding the actual api_reference doc from the toctree. Trying to see why the docs are broken in latest, but this would break them even more without fixing anything.

Reply:
To fix docs:

* move `conf.py` back to `docs` folder, not `docs/source` folder.
* delete docs/source folder
* update readthedocs yaml and makefile to have the right path